lib/manageiq/appliance_console/cli.rb

```diff
@@ -8,6 +8,8 @@ def say(arg)
   end
 end
 
+NETWORK_INTERFACE = "eth0".freeze
```

Comment:
This is a very strange thing to have to do, can you define this constant in `DatabaseReplicationStandby` rather than here?

Reply:
This variable is also defined in appliance_console.rb, that's why I've defined it here though I think  it should be removed in both files.

Reply:
I know, I think you should re-define it in `DatabaseReplication`. That way you don't need to define it here, where it is not used.

Reply:
Yep, makes sense, I'll work in this and all the other proposed changes. Thanks for the review